docs/self-managed/operational-guides/update-guide/850-to-860.md

```diff
@@ -35,3 +35,12 @@ This means:
 Avro schema support is the only breaking change for this Connector. If you do not use Avro schema, you can continue using the Connector as before.
 
 :::
+
+## OpenShift
+
+We added the `global.compatibility.openshift.adaptSecurityContext` variable in the values.yaml that can be used to set the following possible values:
+
+- `force`: all `runAsUser` and `fsGroup` values will be set to null.
+- `disabled`: this is the default value. In this case, the `runAsUser` and `fsGroup` values will not be modified.
```

Comment:
- `force`: The `runAsUser` and `fsGroup` values will be null in all components.
- `disabled`: The `runAsUser` and `fsGroup` values will not be modified (default).

Reply:
done